neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -24,18 +24,49 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation: str='RELU',
```

Comment:
jestli je to kvůli konfiguraci, tak by se to podle mě mělo řešit helper funkcí v config.utils,
která by buď dělala ten switch, nebo něco chytřejšího.